CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 The release log for BoTorch.
 
+## [0.7.2] - Sep 27, 2022
+
+#### New Features
+* A full refactor of `fit_gyptorch_model` and broader fitting methods (#1134).
+  * This introduces a new `fit_gpytorch_mll` method that multiple-dispatches
+    on the model type.
+  * Unlike the previous fitting helpers, `fit_gpytorch_mll` does **not** pass
+   `kwargs` to `optimizer` and instead introduces an optional `optimizer_kwargs`
+    argument.
+  * TODO: Discuss changed error handling behavior.
```

Comment:
@j-wilson could you please add a code comment here to updates this and describe the main new features and changes to previous behavior (in particular error handling)? Thanks!